### PR TITLE
Added wishlist

### PR DESCRIPTION
**Pull request by haarsh24**
1. Add item into wishlist
2. Remove item from wishlist

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -43,3 +43,20 @@
...
   display: none;
 }
+.btn.btn-wishlist {
+  padding: 0rem;
+  margin: 0;
+
+  /* border-radius: 15px;
+  background: var(--primary-color);
+  box-shadow: 5px 5px 9px var(--shadow-dark), -5px -5px 9px var(--shadow-light); */
+}
```
Comment (yashpatel1797) [inline src/App.css:53]: remove comments

**Review by yashpatel1797 — APPROVED** [review]
the file structure is good.
code is clean.
design is consistent.

File: src/components/product-card/product-card.jsx [hunk 2]
```diff
@@ -1,56 +1,90 @@
...
-    const [clicked, setClicked]= useState(false)
-    const item = props.singleProduct;
-    console.log(item);
-    const navigate = useNavigate();
-    const {authState:{isLogin, token}}= useAuth();
-    const {cartDispatch} = useCart();
+    const [cartClicked, setCartClicked] = useState(false);
+    const [wishlistClicked, setWishlistClicked] = useState(false);
+  const item = props.singleProduct;
+  console.log(item);
```
Comment (xii-vi) [inline src/components/product-card/product-card.jsx:16]: remove  console log

File: src/components/product-card/product-card.jsx [hunk 3]
```diff
@@ -1,56 +1,90 @@
...
-        console.log("logged in ")
-             cartDispatch({type:"ADD_TO_CART", payload:item})
-             updateCartItem(item,token);
-       }else{       
-       navigate("/login") 
-       }
+  const addToCardHandler = () => {
+    setCartClicked(true);
+    if (isLogin) {
+      console.log("logged in ");
```
Comment (xii-vi) [inline src/components/product-card/product-card.jsx:27]: same here

**Review by xii-vi — APPROVED** [review]
_Go through comments.
remove console log from the code.
Overall code structure is really good._

**Pull request merged**